frontend/src/less/components/searchbox.less

```diff
@@ -1,15 +1,44 @@
 #searchbox {
+    width: @width-searchbox;
+    max-width: 100%;
+    margin: 0 auto;
+    position: relative;
+    @media (max-width: @screen-sm-min) {
+        margin-top: 30px;
+    }
+    @media (max-width: @screen-xs-max) {
+        padding: 0 20px
+        width: 100%;
+    }
+
     span {
         padding-right: 20px;
```

Comment:
@amcatko nooo :) just the styles, there is no span.how span, there is just the span.how :)